stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/ConnectionState.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.getstream.chat.android.offline.model.connection
+package io.getstream.chat.android.client.models
```

Comment:
This is a breaking change. Can we avoid doing that?
Maybe we could leave it as it + deprecate, then create the same enum in client module and just map both types.

Reply:
One of them will be deprecated so it should be clear they should use another version.
I think it's better to avoid this breaking change